spec/features/user_signs_in_spec.rb

```diff
@@ -19,11 +19,53 @@
     expect(page).to have_content('Sign Out')
   end
 
-  scenario 'specify invalid credentials' do
+  scenario 'specify valid email' do
+    user = FactoryGirl.create(:user)
+
     visit new_user_session_path
 
+    fill_in 'Login', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.